includes/class-wc-stripe.php

```diff
@@ -421,6 +421,14 @@ public function update_prb_location_settings() {
 			? $stripe_settings['express_checkout_button_locations']
 			: [];
 		if ( ! empty( $stripe_settings ) && empty( $prb_locations ) ) {
+			// Use existing payment_request_button_locations if it exists.
+			if ( array_key_exists( 'payment_request_button_locations', $stripe_settings ) ) {
+				$stripe_settings['express_checkout_button_locations'] = $stripe_settings['payment_request_button_locations'];
```

Comment:
It feels like we should also unset the old, now-unused setting as part of this migration logic.
```suggestion
				$stripe_settings['express_checkout_button_locations'] = $stripe_settings['payment_request_button_locations'];
				unset( $stripe_settings['payment_request_button_locations'] );
```

Reply:
Makes sense, Thank you!